plugins/jira-dashboard/src/components/JiraTable/columns.tsx

```diff
@@ -4,139 +4,190 @@ import { Issue } from '@axis-backstage/plugin-jira-dashboard-common';
 import Typography from '@mui/material/Typography';
 import { getIssueUrl } from '../../lib';
 import Stack from '@mui/material/Stack';
+import moment from 'moment';
+import Chip from '@mui/material/Chip';
+import Box from '@mui/material/Box';
 
-export const columns: TableColumn[] = [
-  {
-    title: 'Key',
-    field: 'key',
-    highlight: true,
-    type: 'string',
-    width: '15%',
+export const columnKey: TableColumn<Issue> = {
+  title: 'Key',
+  field: 'key',
+  highlight: true,
+  type: 'string',
+  width: '15%',
 
-    render: (issue: Partial<Issue>) => {
-      if (!issue.self || !issue.key) {
-        return null;
-      }
-      return (
-        <Link
-          to={getIssueUrl(issue.self, issue.key)}
-          title="Go to issue in Jira"
+  render: (issue: Partial<Issue>) => {
+    if (!issue.self || !issue.key) {
+      return null;
+    }
+    return (
+      <Link to={getIssueUrl(issue.self, issue.key)} title="Go to issue in Jira">
+        <Box
+          sx={{
+            display: 'flex',
+            alignItems: 'center',
+          }}
         >
           <img
             src={issue.fields?.issuetype.iconUrl}
             alt={issue.fields?.issuetype.name}
             style={{ paddingRight: '15px' }}
           />
           {issue.key}
-        </Link>
-      );
-    },
+        </Box>
+      </Link>
+    );
+  },
+};
+
+export const columnSummary: TableColumn<Issue> = {
+  title: 'Summary',
+  field: 'fields.summary',
+  highlight: true,
+  type: 'string',
+  width: '45%',
+  render: (issue: Partial<Issue>) => {
+    if (!issue.self || !issue.key) {
+      return null;
+    }
+    return (
+      <Link
+        style={{ lineHeight: 1.5 }}
+        to={getIssueUrl(issue.self, issue.key)}
+        title="Go to issue in Jira"
+      >
+        {issue.fields?.summary}
+      </Link>
+    );
   },
-  {
-    title: 'Summary',
-    field: 'fields.summary',
-    highlight: true,
-    type: 'string',
-    width: '40%',
-    render: (issue: Partial<Issue>) => {
-      if (!issue.self || !issue.key) {
-        return null;
-      }
+};
+
+export const columnPriority: TableColumn<Issue> = {
+  title: 'P',
```

Comment:
Like this change! I would probably have made a separate PR for this though, with these specific UX/UI changes. But I'll allow it :)

Reply:
We can add a changeset for the UI changes.

Reply:
> We can add a changeset for the UI changes.

Yes, that is a good idea!

Reply:
Changeset added